index.html

```diff
@@ -1,544 +1,627 @@
+              <a>
+                <span class="icon is-small"><i
+                    class="fab fa-superpowers"></i></span>
+                <span>Our team</span>
+              </a>
+            </li>
+            <li data-target="pane-4" id="4" tabindex="0" role="button">
+              <a>
+                <span class="icon is-small"><i class="fa fa-film"></i></span>
+                <span>Video</span>
+              </a>
+            </li>
+          </ul>
         </div>
 
-        <div class="field">
-          <label class="label">Email</label>
-          <div class="control has-icons-left has-icons-right">
-            <input class="input is-danger" type="email" placeholder="Email input" value="hello@">
-            <span class="icon is-small is-left">
-              <i class="fas fa-envelope"></i>
-            </span>
-            <span class="icon is-small is-right">
-              <i class="fas fa-exclamation-triangle"></i>
-            </span>
+        <div class="tab-content container">
+          <div class="tab-pane is-active" id="pane-1" class="level">
+            <div class="container">
+              <div class="columns features">
+                <div class="column is-6">
+                  <div class="card">
+                    <div class="card-image">
+                      <div class="image">
+                        <img src="./img/tab-1.jpg" class="modal-button"
+                          alt="Hands with drone remote control">
+                      </div>
+                    </div>
+                    <div class="card-content">
+                      <div class="content">
+                        <div>Best customer's story</div>
+                        <p>We have achieved customer satisfaction and income
+                          increase after having implemented accessibility.</p>
+                        <p>That's a real magic!</p>
+                        <a href="/todo" role="button"
+                          class="button is-link modal-button"
+                          data-target="modal-image1">Learn more</a>
+                      </div>
+                    </div>
+                  </div>
+                </div>
+                <div class="column is-6">
+                  <div class="card">
+                    <div class="card-image">
+                      <div class="image">
+                        <img src="./img/tab-2.jpg"
+                          alt="Island with white sand and palms">
+                      </div>
+                    </div>
+                    <div class="card-content">
+                      <div class="content">
+                        <div>Other Best customer's story.</div>
+                        <p>We were not so sure in success but we have had lots of
+                          new customers after implementing accessibility.</p>
+                        <p>We will recommend Best Company to our friends!</p>
+                        <a href="/todo" role="button"
```

Comment:
No need to replace it to link, otherwise modal dialog is not working. It is just a button